video_emotion/t/test_extract_test_data.py

```diff
@@ -1,6 +1,12 @@
 from video_emotion.extract_test_data import generate_log_data
 
+
 def test_generate_log_data_formats_as_expected():
-    data = generate_log_data(500, [{"happy": 10, "angry": 10, "fear": 10}])
-    assert data == "From,To,Emotions\n0,500,{\'happy\': 10, \'angry\': 10, "\
-                   + "\'fear\': 10}\n"
+    data = generate_log_data(500, [{"happy": 10, "angry": 10, "fear": 10,
+                                    "disgust": 5, "sad": 5, "surprise": 5,
+                                    "neutral": 5}])
+    assert data == ("From,To,"
```

Comment:
Looks correct but tough to read the assertion because of the indentation not on par with the number indentation